spark/src/main/res/values-fr/strings.xml

```diff
@@ -33,12 +33,12 @@
     <plurals name="spark_rating_with_comments_a11y" tools:ignore="UnusedQuantity">
         <item quantity="one">Note de %1$.1f pour %2$d avis</item>
         <item quantity="other">Note de %1$.1f pour %2$d avis</item>
-        <item quantity="many">Note de %1$.1f pour %2$d avis</item>
+        <item quantity="many">Note de %1$.1f pour %2$d d’avis</item>
```

Comment:
Are you sure about these additions of `d'` and `de`?

Reply:
Yeah I [checked french plural rules](https://www.unicode.org/cldr/charts/44/supplemental/language_plural_rules.html#fr) and for the many category it should/must be present